acceptance_modelisation/base_acceptance_map_creator.py

```diff
@@ -10,8 +10,8 @@
 from astropy.time import Time
 from gammapy.data import Observations, Observation
 from gammapy.datasets import MapDataset
-from gammapy.irf import Background2D, Background3D
-from gammapy.irf.background import BackgroundIRF
```

Comment:
I would leave the imports possible at higher level (in irf here) as such. The aim woulld be to import BackgroundIRF with gammapy.irf once we drop gammapy 1.1 support

Reply:
I agree, I will try to clean this.